src/GraphQL/Execution/ExecutionHelper.cs

```diff
@@ -131,6 +136,36 @@ public static Variables GetVariableValues(Document document, ISchema schema, Var
         private static void ThrowNullError(string variableName)
             => throw new InvalidVariableError(variableName, "Received a null input for a non-null variable.");
 
+        private struct VariableName
+        {
+            public VariableName(VariableName variableName, int index)
+            {
+                Name = variableName;
+                Index = index;
+                ChildName = null;
+            }
+            public VariableName(VariableName variableName, string childName)
+            {
+                if (variableName.ChildName == null)
+                {
+                    Name = variableName.Name;
+                    Index = variableName.Index;
+                }
+                else
+                {
+                    Name = variableName;
+                    Index = default;
```

Comment:
Is `default` here `null` or `0` ?

Reply:
null

Reply:
`Nullable<T>` is a struct so I used `default` but if you think `null` is better, I don't care.